public/API/API_GetClaims.php

```diff
@@ -54,7 +55,7 @@
     $claimFilter = ClaimFilters::AllActiveClaims;
 }
 
-$claimsResults = getFilteredClaims(claimFilter: $claimFilter);
+$claimsResults = getFilteredClaims(claimFilter: $claimFilter)->take(1000);
```

Comment:
Suggest passing `limit=1000` instead of fetching all records and then trimming here.

Reply:
Updated in latest.